src/SvelteTooltip.svelte

```diff
@@ -79,6 +79,6 @@
     class:top
     style={style}
   >
-    {tip}
+    {@html tip}
```

Comment:
You're now treating all `tip` attribute content as html which will be undesirable for most cases.

Better to use a `slot` for html and treat the absence of the `tip` attribute as an indication to use the slot

```suggestion
    <slot name="tip">{tip}</slot>
```

Reply:
Thanks. I've implemented custom tooltips with a similiar solution using named slots.